src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h

```diff
@@ -368,6 +351,61 @@ namespace OpenMS
           total_intensity += intensity_sum;
           total_peak_apices += peak_apex_int;
         }
+    
+        if (compute_peak_shape_metrics_)
+        { //for backwards compatibility with TOPP tests
+          // Calculate peak shape metrics that will be used for later QC
+          double width_at_5 = 0;
+          double width_at_10 = 0;
+          double width_at_50 = 0;
+          // double area_2_height = 0;
+          double start_time_at_10 = 0;
+          double start_time_at_5 = 0;
+          double end_time_at_10 = 0;
+          double end_time_at_5 = 0;
+          double total_width = 0;
+          double tailing_factor = 0;
+          double asymmetry_factor = 0;
+          double baseline_delta_2_height = 0;
+          double slope_of_baseline = 0;
+          int points_across_baseline = 0;
+          int points_across_half_height = 0;
+          
+          calculatePeakShapeMetrics_(used_chromatogram, 
```

Comment:
the parameter list is rather long.
consider adding a (class internal) struct
ChromatogramPeakShapeMetrics that contains these as member